### DISCUSSION TITLE
Integrating Hadolint for Dockerfiles

### DISCUSSION THREAD

**Discussion by gbrlgn** [helix-editor/helix#10863] [Ideas] [answered]
I don't really know if there's already a way to do this, but I'd like to have Helix display—as an inlay message, for example—[Hadolint's](https://github.com/hadolint/hadolint) output after saving a Dockerfile so I can interactively tackle any issues regarding its structure without having to exit Helix and run Hadolint separately.

**Comment by kirawi** [comment, marked as answer]
You can try https://github.com/mattn/efm-langserver

**Reply by gbrlgn** [reply]
Seems cool enough for me. Thanks for the answer.
